src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Components/Navbar/user.html.twig

```diff
@@ -0,0 +1,15 @@
+{% import '@SyliusAdmin/Shared/Helper/avatar.html.twig' as avatar %}
+{% import '@SyliusAdmin/Shared/Helper/dropdown.html.twig' as dropdown %}
+
+{% set dropdown_trigger %}
+    <div class="d-flex lh-1 text-reset p-0 cursor-pointer">
+        {% set avatarPath = user.avatar.path|imagine_filter('sylius_admin_admin_user_avatar_thumbnail') %}
+        {{ avatar.small(avatarPath) }}
+        <div class="d-none d-xl-block ps-2">
+            <div class="mb-1 small text-muted">Hello</div>
```

Comment:
For the sake of simplicity for early templates, it can be written as `{{ 'Hello'|trans }}`, so it can be grepped afterwards and replaced with correct message.